git-cms-init

```diff
@@ -328,7 +328,7 @@ if [ -e $CMSSW_GIT_REFERENCE/create-`whoami` ]; then
   (cd $CMSSW_GIT_REFERENCE ; git remote update origin >&${verbose} 2>&1)
 fi
 
-mkdir -p $CMSSW_BASE/src
+[ -e $CMSSW_BASE/src ] || mkdir $CMSSW_BASE/src
```

Comment:
isn't this equivalent to the old one ?

from the description of the PR, I think you want
```bash
[ -d $CMSSW_BASE ] && ! [ -d $CMSSW_BASE/src ] && mkdir $CMSSW_BASE/src
```